pkg/actions/nodegroup/update.go

```diff
@@ -0,0 +1,28 @@
+package nodegroup
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/service/eks"
+	"github.com/kris-nova/logger"
+
+	"github.com/weaveworks/eksctl/pkg/managed"
+)
+
+func (m *Manager) Update() error {
+	ngName := m.cfg.ManagedNodeGroups[0].Name
+	_, err := m.ctl.Provider.EKS().DescribeNodegroup(&eks.DescribeNodegroupInput{
+		ClusterName:   &m.cfg.Metadata.Name,
+		NodegroupName: &ngName,
+	})
+
+	if err != nil {
+		if managed.IsNotFound(err) {
+			return fmt.Errorf("update is only supported for managed nodegroups; could not find one with name %q", ngName)
+		}
+		return err
+	}
+
+	logger.Info("nodegroup successfully updated")
```

Comment:
Where is the update call happening in this func? Am I being blind? Is github diff messing with me? 😂

Reply:
o wait is it happening in https://github.com/weaveworks/eksctl/pull/3855?

Reply:
yeah there's nothing happening here 🙈 all the fun happens over at #3855